From 53d068d2beb0d95f57a5693b779675e595e87223 Mon Sep 17 00:00:00 2001
From: Camelia Groza <camelia.groza@nxp.com>
Date: Tue, 13 Aug 2019 10:45:28 +0800
Subject: [PATCH 017/245] powerpc/mm: include usdpaa header only when built-in

commit 2198a11d3f2faa7e2c1d9c32a4c0fc96d67c83a1 from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux.git

Avoid including the USDPAA header when the USDPAA support isn't built into the kernel.

Signed-of-by: Camelia Groza <camelia.groza@nxp.com>

Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/powerpc/mm/mem.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/powerpc/mm/mem.c b/arch/powerpc/mm/mem.c
index 0650d810f588..72904871872c 100644
--- a/arch/powerpc/mm/mem.c
+++ b/arch/powerpc/mm/mem.c
@@ -38,8 +38,10 @@
 #include <linux/vmalloc.h>
 #include <linux/memremap.h>
 
+#ifdef CONFIG_FSL_USDPAA
 /* See hook_usdpaa_tlb1() */
 #include <linux/fsl_usdpaa.h>
+#endif
 
 #include <asm/pgalloc.h>
 #include <asm/prom.h>
-- 
2.17.1

